api/openapi/specs/api.yaml

```diff
@@ -523,6 +703,34 @@ components:
         name:
           type: string
           example: east
+    DataplaneNetworkingLayout:
+      type: object
+      title: DataplaneLayout
+      description: Dataplane networking layout. It contains information most important information about dataplane and lists of available inbounds and outbounds
+      required: [ kri, name ]
+      properties:
+        kri:
+          type: string
+          example: kri_msvc_mesh-1_us-east-2_kuma-demo_demo-app_http-port
+        name:
```

Comment:
I know it was in the MADR, but maybe we can remove `name` here as it's exactly the same as the name in path. Also we have `kri` right there.